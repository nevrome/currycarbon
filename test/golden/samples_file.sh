currycarbon "3000,30" --samplesFile actual_data/samples_file.tsv -n 3 --seed 123