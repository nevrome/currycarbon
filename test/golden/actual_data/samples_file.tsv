id	yearBCAD
1	-1262
1	-1362
1	-1235
